apps/studio/components/interfaces/Database/Functions/CreateFunction.tsx

```diff
@@ -1,478 +1,408 @@
+                    />
+                  </FormControl_Shadcn_>
+                </FormItemLayout>
+              )}
+            />
+            {!isEditing && (
+              <FormField_Shadcn_
+                control={form.control}
+                name="return_type"
+                render={({ field }) => (
+                  <FormItemLayout
+                    label="Return type"
+                    description="Tables made in the table editor will be in 'public'"
+                    layout="horizontal"
+                  >
+                    <FormControl_Shadcn_>
+                      <Listbox
+                        size="small"
+                        value={field.value}
+                        onChange={(value) => field.onChange(value)}
+                      >
+                        <Listbox.Option value="void" label="void">
+                          void
+                        </Listbox.Option>
+                        <Listbox.Option value="record" label="record">
+                          record
+                        </Listbox.Option>
+                        <Listbox.Option value="trigger" label="trigger">
+                          trigger
+                        </Listbox.Option>
+                        <Listbox.Option value="integer" label="integer">
+                          integer
+                        </Listbox.Option>
+                        {POSTGRES_DATA_TYPES.map((x: string) => (
+                          <Listbox.Option key={x} value={x} label={x}>
+                            {x}
+                          </Listbox.Option>
+                        ))}
+                      </Listbox>
+                    </FormControl_Shadcn_>
+                  </FormItemLayout>
                 )}
-              </div>
+              />
             )}
-          </div>
-        </CreateFunctionContext.Provider>
-        <ConfirmationModal
-          visible={isClosingPanel}
-          header="Discard changes"
-          buttonLabel="Discard"
-          onSelectCancel={() => setIsClosingPanel(false)}
-          onSelectConfirm={() => {
-            setIsClosingPanel(false)
-            setVisible(!visible)
-          }}
-        >
-          <Modal.Content>
-            <p className="py-4 text-sm text-foreground-light">
-              There are unsaved changes. Are you sure you want to close the panel? Your changes will
-              be lost.
-            </p>
-          </Modal.Content>
-        </ConfirmationModal>
-      </SidePanel>
-    </>
+          </SidePanel.Content>
+          <SidePanel.Separator />
+          <SidePanel.Content>
+            <FormFieldArgs readonly={isEditing} />
+          </SidePanel.Content>
+          <SidePanel.Separator />
+          <SidePanel.Content>
+            <FormField_Shadcn_
+              control={form.control}
+              name="definition"
+              render={({ field }) => (
+                <FormItemLayout
```

Comment:
This is maybe a good example of where not to use FormItemLayout.
Unless we actually start supporting a bigger <h5> esque label style as an option.